### PR TITLE
docs: upload folder drag and drop

### PR DESCRIPTION
**Pull request by sissbruecker**
Document that upload now supports drag and drop of folders.

### CODE REVIEW

File: articles/components/upload/index.adoc [hunk 1]
```diff
@@ -69,6 +69,9 @@ include::{root}/frontend/demo/component/upload/react/upload-drag-and-drop.tsx[re
 endif::[]
 --
 
+Upload also allows to drag & drop folders. This recursively uploads all files from the folder and its sub-folders. Note that the folder structure will be flattened and the names of uploaded files will not indicate in which folder they were in. Uploading folders is currently only supported when using drag & drop, but not when using the file picker.
```
Comment (rolfsmeds) [inline articles/components/upload/index.adoc:72]:
```suggestion
Upload can be configured to allow drag & drop uploading of folders. This uploads all files from the folder and its sub-folders recursively. Note that the folder structure will be flattened and the names of uploaded files will not indicate in which folder they were in. Uploading folders is currently only supported by drag & drop, not through the native file picker opened through the upload button.
```

**Review by rolfsmeds — CHANGES_REQUESTED** [review]

File: articles/components/upload/index.adoc (hunk 1 above)
Comment (sissbruecker) [reply articles/components/upload/index.adoc:72]: @rolfsmeds We don't have a config option for this, it just works out of the box.

Comment (rolfsmeds) [reply articles/components/upload/index.adoc:72]: Ah, indeed, lemme rephrase it.

```suggestion
Upload supports uploading folders via drag & drop. This uploads all files from the folder and its sub-folders recursively. Note that the folder structure will be flattened and the names of uploaded files will not indicate in which folder they were in. Uploading folders is currently not supported through the native file picker.
```

Comment (sissbruecker) [reply articles/components/upload/index.adoc]: Thanks!

**Review by rolfsmeds — APPROVED** [review]

**Pull request merged**